src/modules/account/components/signin-button/index.tsx

```diff
@@ -17,13 +18,7 @@ const AccountSignInButton: React.FC<Generic> = ({ className }) =>
           className
         )}
       >
-        <UserCircleIcon
-          aria-hidden="true"
-          className={tw(
-            "size-6 flex-shrink-0",
-            "text-light-secondary dark:text-dark-secondary group-hover:text-light-text/60 dark:group-hover:text-dark-text/60"
-          )}
-        />
+        <Icon icon={<UserCircleIcon />} />
```

Comment:
You forgot to put back the classes that already had the icon before.

```suggestion
       <Icon
          icon={<UserCircleIcon />}
          className={tw(
            "size-6 flex-shrink-0",
            "group-hover:text-light-text/60 dark:group-hover:text-dark-text/60"
          )}
        />
```